libs/components/datetime/src/lib/modules/date-range-picker/date-range-picker.component.spec.ts

```diff
@@ -466,6 +467,18 @@ describe('Date range picker', function () {
     expect(component.numValueChangeNotifications).toEqual(1);
   }));
 
+  it('should set the calculator ID to the default on initialization', fakeAsync(() => {
+    fixture.detectChanges();
+
+    expect(getCalculatorSelect().value).toEqual(
+      `${SkyDateRangeCalculatorId.AnyTime}`,
```

Comment:
Can we have this just check that it is '0' jic we change what the 0th calculator name is in the future?

Reply:
We should test against the enum name, since that's the thing that will always remain the same (we might change the underlying number value, but it will always be set to `SkyDateRangeCalculatorId.AnyTime` as the default, as far as the consumer is concerned).

Reply:
oh neat :o TIl some implementation details for DRP